static/js/settings.js

```diff
@@ -502,6 +502,30 @@ function _setup_page() {
         });
     });
 
+    $("#emoji_alt_code").change(function () {
+        var emoji_alt_code = this.checked;
+        var data = {};
+        data.emoji_alt_code = JSON.stringify(emoji_alt_code);
+        var context = {};
+        if (data.emoji_alt_code === "true") {
+            context.side = i18n.t('enabled');
+        } else {
+            context.side = i18n.t('disabled');
```

Comment:
probably want to call this variable something more accurately descriptive than "side".  Maybe `context.enabled_or_disabled`?  I'm thinking about what would make this clearest to translators in the next block.